src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+}
```

Comment:
make a list of tags you want reset margins for, do not use `*`

Reply:
**Requirements:
reset browser default margins**

So i don`t need to do this?